lms/djangoapps/teams/models.py

```diff
@@ -48,9 +49,11 @@ def create(cls, name, course_id, description, topic_id=None, country=None, langu
         """
 
         team_id = generate_unique_readable_id(name, cls.objects.all(), 'team_id')
+        discussion_id = 'team:' + team_id
```

Comment:
To make this more likely to not being a user-defined name, I'd suggest prefixing with team_id instead of team.